standalone/src/DelphesRootReader.h

```diff
@@ -18,6 +18,7 @@
 #include "ExRootAnalysis/ExRootTreeWriter.h"
 
 #include <iostream>
+#include <sstream>
```

Comment:
This one might be missing in other readers too probably.

Reply:
apple-clang was happy with this patch, but I can add it elsewhere too.

Reply:
Ah then maybe something else pulls it in, or it is maybe already there (or something pulls it in transitively). But since we are here already, might as well check.